kore/src/Log.hs

```diff
@@ -266,12 +263,11 @@ instance Monad m => MonadLog (LoggerT m) where
         lift $ someLogAction <& toEntry entry
     {-# INLINE logEntry #-}
 
-    logWhile !entry2 action = do
+    logWhile entry2 action = do
```

Comment:
Why is `entry2` no longer strict?

Reply:
We want the entries to be evaluated lazily. This probably wasn't a problem in practice, because all the entries seem to have lazy fields, but it is just a precaution.